ooni/scripts/ooniprobe.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import webbrowser
+from multiprocessing import Process
+
+from twisted.internet import task, defer
+
+def ooniprobe(reactor):
+    from ooni.ui.cli import runWithDaemonDirector, runWithDirector
+    from ooni.ui.cli import setupGlobalOptions, initializeOoniprobe
+
+    global_options = setupGlobalOptions(logging=True, start_tor=True,
+                                        check_incoherences=True)
+    if global_options['queue']:
+        return runWithDaemonDirector(global_options)
+    elif global_options['initialize']:
```

Comment:
In this chain of `if..elif` you can omit the `elif` and go for `if` because you `return`